<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaEmprestimo</class>
 <widget class="QDialog" name="TelaEmprestimo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitulo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tela Emprestimo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboUsuario">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>79</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsuario">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labeilLivro">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Livro: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboLivro">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>79</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="dataEmprestimo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>272</width>
     <height>149</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addLivro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar Livro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUser">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar Usuário</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="novoLivro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="novoUser">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addEmprestimo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Emprestimo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboEmprestimo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="devolver">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Devolver</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmprestimo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Empréstimo:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TelaEmprestimo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TelaEmprestimo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
